src/ExecutionContext.re

```diff
@@ -21,4 +21,6 @@ external name: t => string = "";
  * prototype. Returns a handle to an array of objects with this prototype.
  */
 [@bs.send]
-external queryObjects: (t, ~prototypeHandle: JSHandle.t) => JSHandle.t = "";
+external queryObjects:
+  (t, ~prototypeHandle: JSHandle.t('a)) => JSHandle.t(array(Js.t({..}))) =
```

Comment:
The inner return type here can be any object. Not sure typing it as a `Js.t` is appropriate. Maybe it should be left as `'b` a just let the user specify the type rather than possibly forcing them to cast.